gatsby-node.js

```diff
@@ -1,7 +1,60 @@
-/**
- * Implement Gatsby's Node APIs in this file.
- *
- * See: https://www.gatsbyjs.org/docs/node-apis/
- */
+const fs = require('fs')
+const path = require('path')
+const { transformResults } = require('./src/transformers/results')
 
-// You can delete this file if you're not using it
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  const { data } = await graphql(`
```

Comment:
Eu vi que `persons` foi usada mais de uma vez, mas que existe uma outra variável nomeada `persons`

Você pode desestruturar essa `persons` atribuindo um novo nome para identificá-la

```suggestion
  const { data: { persons: searchedPersons } } = await graphql(`
```